spec/spec_helper.rb

```diff
@@ -33,10 +33,10 @@ def image_path(path, options = {})
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
-  config.expect_with :rspec do |c|
+  config.expect_with(:rspec) do |c|
```

Comment:
At first, I tried to include everything except for `*_spec.rb`, but that was too optimistic, so I changed the config to exclude `spec/*`, but didn't remove the changes from this PR yet.